tests/semantic_analyser.cpp

```diff
@@ -608,6 +608,13 @@ TEST(semantic_analyser, call_stack)
   test("kprobe:f { @x = 3; ustack(perf, @x) }", 10);
 }
 
+TEST(semantic_analyser, call_fd_path)
+{
+  test("kprobe:f { $k = fd_path( 1 ) }", 0);
+  test("kretprobe:f { $k = fd_path( 1 ) }", 0);
+  test("tracepoint:category:event { $k = fd_path( 1 ) }", 0);
```

Comment:
would be good to test some failures too `fd_path(-100)` `fd_path("abc")`. 

And `fd_path(arg0)` as that is valid too.

Reply:
ok